client/src/document/organisms/sidebar/index.tsx

```diff
@@ -3,18 +3,25 @@ import { Link } from "react-router-dom";
 
 import "./index.scss";
 
+function SidebarContainer({ children }) {
+  return (
+    <aside id="sidebar-quicklinks" className="sidebar">
+      {children}
+    </aside>
+  );
+}
+
 export function RenderSideBar({ doc }) {
-  console.log(doc);
   if (!doc.related_content) {
     if (doc.sidebarHTML) {
       return (
-        <aside
-          id="sidebar-quicklinks"
-          className="sidebar"
-          dangerouslySetInnerHTML={{
-            __html: `<h4>Related Topics</h4>${doc.sidebarHTML}`,
-          }}
-        />
+        <SidebarContainer>
+          <span
+            dangerouslySetInnerHTML={{
+              __html: `<h4>Related Topics</h4>${doc.sidebarHTML}`,
+            }}
+          ></span>
+        </SidebarContainer>
```

Comment:
What's wrong with:
```jsx
<SidebarContainer>
  <h4>Related Topics</h4>
  <div dangerouslySetInnerHTML={{
              __html: `<h4>Related Topics</h4>${doc.sidebarHTML}`,
            }}/>
</SidebarContainer>
```

Reply:
The extra `<div>` isn't wonderful but if the CSS depends on immediate `aside + div` then can't we fix that instead?

Reply:
Nope, that is not a problem. We should definitely never rely on such direct nesting.

Reply:
> What's wrong with:

Absolutely nothing :) That works as well